test/test-PerfMeter.js

```diff
@@ -0,0 +1,162 @@
+/* globals Services */
+
+"use strict";
+const simplePrefs = require("sdk/simple-prefs");
+
+const {before, after} = require("sdk/test/utils");
+const tabs = require("sdk/tabs");
+const {setTimeout} = require("sdk/timers");
+const {ActivityStreams} = require("lib/ActivityStreams");
+const {Cu} = require("chrome");
+Cu.import("resource://gre/modules/Services.jsm");
+
+let ACTIVITY_STREAMS_URL;
+let app;
+
+function openTestTab(notifyEvent) {
+  let tabData = {};
+  tabData.showPromise = new Promise(resolve => {
```

Comment:
Because I need the tab object to close it, and you never know which tab was closed when in the situation of multiple tabs being open simultaneously